manifests/quick-start/mysql/overlays/workflow-controller-configmap.yaml

```diff
@@ -13,6 +13,10 @@ data:
         secretKeySecret:
           name: my-minio-cred
           key: secretkey
+    metricsConfig:
+      enabled: true
+      path: /metrics
+      port: 9090
```

Comment:
Can we match the example please? Options:

1. Run on 8080 to match the example.
2. Change both to run on another port. Maybe we should use 2090 for metrics and 2091 for telemetry? @jessesuen ?

But please don't use port 9090! Developers would probably want to run Prometheus on its default port.